build.sbt

```diff
@@ -282,8 +321,6 @@ lazy val readme = scalatex.ScalatexReadme(
 
 lazy val sharedSettings = Def.settings(
   scalaVersion := ScalaVersion,
-  crossScalaVersions := ScalaVersions,
-  crossVersion := CrossVersion.binary,
```

Comment:
How about we keep these lines in place and just override these settings in scalahostSbt? It looks like this will significantly reduce the amount of required changes.